command/rootchain/staking/params.go

```diff
@@ -0,0 +1,46 @@
+package staking
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/0xPolygon/polygon-edge/command/helper"
+	sidechainHelper "github.com/0xPolygon/polygon-edge/command/sidechain"
+)
+
+const (
+	stakeManagerFlag = "stake-manager"
+)
+
+type stakeParams struct {
+	accountDir       string
+	accountConfig    string
+	stakeManagerAddr string
+	jsonRPC          string
+	amount           uint64
+	chainID          uint64
+}
+
+func (v *stakeParams) validateFlags() error {
+	return sidechainHelper.ValidateSecretFlags(v.accountDir, v.accountConfig)
```

Comment:
Maybe validate url with `ParseJSONRPCAddress(jsonrpcAddress string) (*url.URL, error)` from `command/helper`

Reply:
We are not validating this in any command :confused:

Reply:
Done: https://github.com/0xPolygon/polygon-edge/pull/1408/commits/8da5799891a0c9d9dad553555b4f6a1893d2086c